src/hooks.ts

```diff
@@ -0,0 +1,6 @@
+import type { Handle } from '@sveltejs/kit';
+
+export const handle: Handle = async ({ event, resolve }) => {
+  const response = await resolve(event, { ssr: false });
```

Comment:
```suggestion
  return await resolve(event, { ssr: false });
```
or even without await